src/parser/rogue/outlaw/modules/spells/Dispatch.js

```diff
@@ -18,7 +18,7 @@ class Dispatch extends Analyzer {
   constructor(...args) {
     super(...args);
     const hasRelevantTrait = this.selectedCombatant.hasTrait(SPELLS.ACE_UP_YOUR_SLEEVE.id) || this.selectedCombatant.hasTrait(SPELLS.DEADSHOT.id);
```

Comment:
Are these Azerite Traits? We are removing Azerite Traits and BFA items for Prepatch. So if they are traits, you might want to just remove them

Reply:
havent answered this ... are these Azerite Traits??

Reply:
I've removed these multiple times. My guess is that they keep coming back when I merge the shadowlands branch into this to make it merge compatible.